### DISCUSSION TITLE
Daemonset for fluentd 1.17.0 forthcoming?

### DISCUSSION THREAD

**Discussion by einheber-seed** [fluent/fluentd#4518] [General]
Fluentd 1.17.0 was released in April.  Is there a corresponding daemonset forthcoming?

**Comment by kenhys** [comment]
fluent-plugin-kubernetes_metadata_filter doesn't support Fluentd v1.17.0 yet, so after newer version of fluent-plugin-kubernets_metadata_filter is released,  daemonset image will be shipped.

**Comment by skforever99** [comment]
Please let us know when we can expect the " fluentD 1.17.0 Daemonset image " ?

**Comment by kenhys** [comment]
It depends on upstream (fluent-plugin-k8s metadata_filter ) developer's decision, so we can't forecast "when?".

Already sent a feedback, but It seems that it has pended.
https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/pull/382
